src/pages/PlacePage.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="test-page">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <div class = "searchbar">
+      <center>
+      <input class="text-search" type="text" v-model="search" placeholder="Nama Tempat Yang dicari" @change="searching" >
+      <input class="button" type="submit" value ="cari">
```

Comment:
You need to decide what event that you're going to used. @change or @click?

Because if @change, submit button is not necessary anymore since the input already handle the method. But if @click, you have to remove @change event from the input, and put @click event in the submit button